common/openapi.yml

```diff
@@ -114,44 +118,23 @@ paths:
         500:
           $ref: '#/components/responses/ServerError'
 
-  /api/pipelines/v1/{pipelineName}/jobs:
-    parameters:
-      - $ref: '#/components/parameters/PipelineName'
-      - $ref: '#/components/parameters/Limit'
-      - $ref: '#/components/parameters/PageToken'
-    get:
-      summary: Retrieve all pipeline-specific jobs requested by user
-      operationId: getPipelineJobs
-      tags: [ pipelines ]
-      responses:
-        200:
-          description: Success
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/GetJobsResponse'
-        400:
-          $ref: '#/components/responses/BadRequest'
-        500:
-          $ref: '#/components/responses/ServerError'
-
-  /api/pipelines/v1/{pipelineName}/result/{jobId}:
+  /api/pipelineruns/v1/{pipelineName}/result/{jobId}:
     parameters:
       - $ref: '#/components/parameters/PipelineName'
       - $ref: '#/components/parameters/JobId'
     get:
-      summary: Retrieve result for a specified pipeline job
-      operationId: getPipelineJobResult
-      tags: [ pipelines ]
+      summary: Retrieve result for a specified pipeline run
+      operationId: getPipelineRunResult
+      tags: [ pipelineRuns ]
       responses:
         200:
-          description: Job is complete (succeeded or failed)
+          description: PipelineRun is complete (succeeded or failed)
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/CreateJobResponse'
+                $ref: '#/components/schemas/CreatePipelineRunResponse'
```

Comment:
a little off topic but i wonder if we want to rename the `CreatePipelineRunResponse` schema.  It makes sense for the create pipeline api call but not like this result call.  Maybe `AsyncPipelineRunResponse` ? Curious what you think